internal/casing/camel.go

```diff
@@ -50,6 +55,18 @@ func Camel(s string) string {
 	return string(t)
 }
 
+// CamelIdentifier returns the CamelCased identifier without affecting the package name/path if any.
+func CamelIdentifier(s string) string {
+	dot := "."
```

Comment:
Nit: could probably make this `const`.
```suggestion
	const dot = "."
```

Reply:
Done